shared/src/jvmMain/kotlin/Music.jvm.kt

```diff
@@ -1,36 +1,30 @@
 import java.io.File
 
-var SongExtensions = listOf("mp3", "wav", "flac", "opus", "m4a")
+val SongExtensions = listOf("mp3", "wav", "flac", "opus", "m4a")
```

Comment:
Let's add all the media formats supported by all the three media player libraries. Maybe we can add it in the next PR

Reply:
Yeah this is definitely not ideal, but we can figure that out later